.zenodo.json

```diff
@@ -0,0 +1,30 @@
+{
+    "creators": [
+        {
+            "orcid": "0000-0001-7782-6909",
+            "name": "Karimi-Asli, Kaveh"
+        },
+        {
```

Comment:
There are two Lasse here! :)

Reply:
haha! I used him as template and then got confused. Fixing it now